dist/index.html

```diff
@@ -31,18 +31,16 @@
             </p>
             <p class="subtitle">Work in progress.</p>
             <p class="heading-small">Contribute</p>
-            <p class="subtitle">
-                You can use these platforms to communicate/contribute with us.
-            </p>
+            <p class="subtitle">Use these platforms to communicate/contribute with us.</p>
             <div class="logos">
-                <a href="https://github.com/codidact" class="btn-link">
-                    <img class="invert" alt="Github logo" src="assets/img/github-logo.png" />
+                <a href="https://github.com/codidact" class="btn-link" title="GitHub">
+                    <img src="assets/img/github-logo.png" class="invert" alt="GitHub logo" />
                 </a>
-                <a href="https://discord.gg/WZ7aTst" class="btn-link">
-                    <img alt="Discord logo" src="assets/img/discord-logo.png" />
+                <a href="https://discord.gg/WZ7aTst" class="btn-link" title="Discord (chat)">
+                    <img src="assets/img/discord-logo.png" alt="Discord logo" />
                 </a>
-                <a href="https://forum.codidact.org/" class="btn-link">
-                    <img alt="Discourse logo" src="assets/img/discourse-logo.png" />
+                <a href="https://forum.codidact.org/" class="btn-link" title="Discourse (forum)">
+                    <img src="assets/img/discourse-logo.png" alt="Discourse logo" />
                 </a>
             </div>
         </div>
```

Comment:
The changes in this file look okay. Adding the title is a nice thing, because not everyone knows the logos. :+1: